ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -435,10 +435,14 @@ public struct Avatar: View, TokenizedControlView {
         /// - Returns: hash code of string
         private static func javaHashCode(_ text: NSString) -> Int32 {
             var hash: Int32 = 0
-            for i in 0..<text.length {
-                // Allow overflows, mimicking Java behavior
-                hash = 31 &* hash &+ Int32(text.character(at: i))
-            }
+            var len = text.length - 1
```

Comment:
might be worth cross referencing where this code comes from?

Reply:
How do I do that?

Reply:
comment to the web(Fluent react) file?

Reply:
I think I already did that in the description, should I do it in the code as well?

Reply:
i think so. also make sure the link is the actual file not the link reference to a commit by a particular user